runtime/doc/various.txt

```diff
@@ -14,11 +14,13 @@ Various commands					*various*
 
 							*CTRL-L*
 CTRL-L			Clear and redraw the screen.  The redraw may happen
-			later, after processing typeahead.
+			later, after processing typeahead. Terminal buffers
+			fit the embedded terminal into the current window.
 
 							*:redr* *:redraw*
 :redr[aw][!]		Redraw the screen right now.  When ! is included it is
-			cleared first.
+			cleared first and terminal buffers fit the embedded
+			terminal into the current window.
```

Comment:
here and above, instead of "fit the embedded terminal into the current window", a bit more direct/compact wording could be "visible terminals are resized to their windows". (Just to confirm: non-visible terminals should not be resized, right?)

By the way: if a terminal is shown in multiple windows, which one takes precedence (incl. if none are focused)? I think it should choose the focused window, or the biggest window if none are focused.